data/pathfinder/legendary_games/ap_plugins/treasury_of_the_pharaohs/_treasury_ot_pharaohs.pcc

```diff
@@ -0,0 +1,26 @@
+# CVS $Revision a.1$ $Author Axxroy$ -- Sun Feb 25 14:45:00 2017 -- reformated by prettylst.pl v1.35 (build 553)
+CAMPAIGN:Legendary - Treasury of the Pharaohs
+GAMEMODE:Pathfinder
+RANK:9
+GENRE:Fantasy
+BOOKTYPE:Supplement
+SETTING:Pathfinder
+TYPE:Legendary Games.Adventure Path Add-ons.Egyptian
+PRECAMPAIGN:6,INCLUDESBOOKTYPE=Core Rules,INCLUDES=Ultimate Combat,INCLUDES=Advanced Player's Guide,INCLUDES=Ultimate Equipment,INCLUDES=Ultimate Magic,INCLUDES=Melee Tactics Toolbox
+PUBNAMELONG:Legendary Games
+PUBNAMESHORT:Legendary
+PUBNAMEWEB:http://www.makeyourgamelegendary.com/
+SOURCELONG:Treasury of the Pharaohs
+SOURCESHORT:TotP
+SOURCEWEB:http://www.makeyourgamelegendary.com/
+SOURCEDATE:2015-04
+ISOGL:YES
+COPYRIGHT:Treasury of the Pharaohs, Copyright 2015 by Legendary Games
```

Comment:
Customary to include all of the OGL Copyrights in this section. The final one is the PCGen which you would include yourself as a Data Monkey. (Need to get your credit in here).